### PR TITLE
Update the triton_account data source.

### PR DESCRIPTION
**Pull request by kwilczynski**
This commit updates the `triton_account` data source with the following changes:

  - Adds more logging at the debug level
  - Adds more comments to the code
  - Adds new `login` and `email` attributes to the data source output
  - Updates error messages
  - Updates documentation
  - Removes an underscore from the variable name in the acceptance test

Signed-off-by: Krzysztof Wilczynski <kw@linux.com>

### CODE REVIEW

**Comment by kwilczynski** [comment]
Hi @cheapRoc!

This is first Pull Request in a series of data source updates. More coming!

**Comment by kwilczynski** [comment]
Acceptance tests:

```
$ make testacc TEST=./triton/ TESTARGS='-run=TestAccTritonAccount'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./triton/ -v -run=TestAccTritonAccount -timeout 120m
=== RUN   TestAccTritonAccount
--- PASS: TestAccTritonAccount (5.38s)
PASS
ok  	github.com/terraform-providers/terraform-provider-triton/triton	5.402s
```

**Comment by kwilczynski** [comment]
Changes to the documentation:

localhost_4567_docs_providers_triton_d_triton_account html

File: triton/data_source_account.go [hunk 1]
```diff
@@ -19,20 +30,28 @@ func dataSourceAccount() *schema.Resource {
...
+// dataSourceAccountRead retrieves details about current Account from Triton
+// using the Account API. The current Account name will be the same as the
+// one currently configured in the Triton provider.
 func dataSourceAccountRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*Client)
+
 	c, err := client.Account()
 	if err != nil {
-		return err
+		return errwrap.Wrapf("Error creating Account client: {{err}}", err)
```
Comment (kwilczynski) [inline triton/data_source_account.go]: I got feedback not to use `errwrap.Wrapf`, do we want to remove error wrapping completely or use `errors.Wrap` instead?

Comment (kwilczynski) [reply triton/data_source_account.go]: Fixed.

**Comment by cheapRoc** [comment]
@kwilczynski If this is completely ready for merge I can fire away.

**Comment by kwilczynski** [comment]
Hi @cheapRoc, thank you for the review!

It's good to go! 👍

**Comment by cheapRoc** [comment]
Thanks for another contribution!

**Pull request merged**